### ISSUE TITLE
program crash when start

### ISSUE THREAD

**Issue by wjx0912** [andrepolischuk/keep#52]
i have test on 2 pc, both crash when start, both win10.

![screenshot](http://185.239.71.40/keep_crash.png)

can help fix it, thanks.

**Comment by e-t-l** [comment]
I had the exact same problem when I tried to run v0.1.9.

**Comment by FabioMantovani** [comment]
Hi Andrey.
I used your Keep-windows-0.1.5 for many years: so usefull!!! -> thank you.
I installed the new (v0.1.9) on Wn 8.1 and I got the same message of wjx0912:
"A javascript error occurred in the main process".
Please, could you fix the problem?
In case, I'm available to test it with you.
Thank you.
Kind regards.

**Comment by e-t-l** [comment]
Is there an installation package for v0.1.5 (for Windows)? I checked the Release history but only found a zip folder and couldn't figure out how to install it from there. I'd like to rollback to v0.1.5 as a temporary solution but can't figure out how to install that older version.

**Comment by FabioMantovani** [comment]
You need to download the zip file, unzip it and then run Keep.exe.
OK: it doesn't require installation. It is standalone code. But it worked wonderfully until about 2 weeks ago.
Now it doesn't work: Google doesn't accept the login procedure :(
We need an help from Andrey :)

**Comment by Raj-Joshi-dev** [comment]
Hi, I wanted to try this awesome app but unfortunately, there's this Javascript error. 

OS: Linux Mint 20
File type: AppImage 

![Screenshot from 2020-08-31 15-28-16](https://user-images.githubusercontent.com/52413491/91725268-c888f380-eb9e-11ea-9371-0a46e54da8b1.png)

I would appreciate a fix. Thank you

**Comment by andrepolischuk** [comment]
sorry it doesn't work, I'll take a look this week

**Comment by codeMoney095** [comment]
Having the same issue on Manjaro 20. Thanks for your work, I would really like to try it out though!

**Comment by FabioMantovani** [comment]
Hi Andrey.
I installed the latest version (21 Aug), but I see again the same messsage.
Please, when you have a bit of time, could you have a look about it?
Thank you so much.
Kind regards.
![Senza titolo-1](https://user-images.githubusercontent.com/47545804/94249864-c48f8d80-ff20-11ea-9fb6-6ac535d3cac0.jpg)

**Comment by MaverickMartyn** [comment]
For some reason the build folder isn't being included in the packaged app.asar file.
A workaround, until a fix is released, is as follows:
1. Open `%localappdata%\Programs\keep\resources`
2. Open app.asar using 7Zip or a similar tool.
3. Create a new folder inside app.asar named `build`.
4. Download the files from the build folder in the repository, found at https://github.com/andrepolischuk/keep/tree/master/build
5. Drag all three icon files into the newly created build folder inside app.asar.
6. ???
7. Profit!

**Comment by e-t-l** [comment]
@MaverickMartyn Is there something I'm missing? I tried as you suggested but couldn't open app.asar with 7Zip (or Peazip, for that matter). Got the error message: "Couldn't open app.asar as archive." When I tried converting it to a different archive filetype then opening, got the error message that it wasn't a readable archive. This is all from a fresh install of v0.1.9 btw.

**Comment by FabioMantovani** [comment]
@MaverickMartyn : thank you very much, it works!
@e-t-l : I got the same problem. Please, follow these steps:
- download 7-Zip
- follow these instructions: https://www.tc4shell.com/en/7zip/asar/
- then you can open the .asar folder and you can follow the Maverick's instructions.
Cheers.

**Comment by e-t-l** [comment]
@FabioMantovani @MaverickMartyn Alright I did as you said, used tc4shell to add the build folder inside app.asar, with those three files inside of it... and I still get the same javascript error when I try to run Keep.

**Comment by MaverickMartyn** [comment]
@e-t-l I just got the issue fixed in my branch, so hopefully @andrepolischuk will be able to put out a release ASAP. :)
After doing a bit more digging, the error is caused by `icon: join(__dirname, '../build/icon.png'),` beign unable to find the missing icon files. A minor change to package.json allows the icons to be included properly in app.asar, fixing the issue.
In my own test it loads fine. Even though it's only about one line, I took the liberty of creating a pull request, since I am fiddling around with some other stuff anyways. See PR: #53
